app/templates/buckets/list.html

```diff
@@ -21,10 +21,12 @@
           <td>{{ bucket.id }}</td>
           <td><a href="{{ url_for('buckets.details', {id: bucket.id}) }}">{{ bucket.name }}</a></td>
           <td class="align-right">
-            <form action="{{ url_for('buckets.delete', {id: bucket.id}) }}" method="post" class="inline-form">
-              <input type="hidden" id="redirect" name="redirect" value="{{ url_for('buckets.list') }}">
-              <input type="submit" class="button button-secondary right js-confirm" value="Delete bucket" data-confirm-message="Are you sure you want to delete this bucket?">
-            </form>
+            {% if current_user_is_bucket_admin or current_user.is_superuser %}
```

Comment:
Maybe it's a stupid question but what is `current_user_is_bucket_admin` in the context of the list of apps here?

Reply:
This is the list of buckets. `current_user_is_bucket_admin` is determining if the logged in user has the right to manage or delete each bucket.

Reply:
Isn't being an admin of a bucket something depending on the specific bucket? As far as I can tell this flag doesn't change at each flag iteration.

(Not a massive deal anyway as permissions would be enforced in the API regardless)